docs/upgrade-notes.asciidoc

```diff
@@ -0,0 +1,1714 @@
+[[breaking-changes-summary]]
+== Upgrade notes
+
+////
+USE THE FOLLOWING TEMPLATE to add entries to this document.
+1. Copy from "[discrete]" to the last "====" included.
+2. Paste and edit the template in the right area section. Most recent entries should be at the top of the section. 
+3. Make sure to edit the anchor ID [[REPO-PR]] with proper values, then add the corresponding entry to the doc link service https://github.com/elastic/kibana/blob/main/packages/kbn-doc-links/src/get_doc_links.ts:
+
+codeId: `${KIBANA_DOCS}breaking-changes-summary.html#REPO-PR` // replace `codeId` with the ID of your choice that you want to call in the Upgrade Assistant code, and replace `REPO-PR` with the corresponding values you edited in this document.
+
+[discrete]
+[[REPO-PR]]
+.[FEATURE] TITLE TO DESCRIBE THE CHANGE. (VERSION)
+[%collapsible]
+====
+*Details* +
+ADD MORE DETAILS ON WHAT IS CHANGING AND A LINK TO THE PR INTRODUCING THE CHANGE
+
+*Impact* +
+ADD INFORMATION ABOUT WHAT THIS CHANGE WILL BREAK FOR USERS
+
+*Action* +
+ADD INSTRUCTIONS FOR USERS LOOKING TO UPGRADE. HOW CAN THEY WORK AROUND THIS?
+====
+////
+
+This page summarizes all breaking changes and deprecation notices introduced to {kib} version 8 since 8.0. Review these changes before upgrading Kibana to mitigate the impact to your application.
+
+NOTE: For the complete Elastic Security solution release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+[float]
+=== Breaking changes
+
+[float]
+==== Kibana APIs
+
+[discrete]
+[[breaking-162506]]
+.Get case metrics APIs became internal. (8.10)
+[%collapsible]
+====
+*Details* +
+The get case metrics APIs are now internal. For more information, refer to ({kibana-pull}162506[#162506]).
+====
+
+[discrete]
+[[breaking-155470]]
+.Removed legacy project monitor API. (8.8)
+[%collapsible]
+====
+*Details* +
+The project monitor API for Synthetics in Elastic Observability has been removed. For more information, refer to {kibana-pull}155470[#155470].
+
+*Impact* +
+In 8.8.0 and later, an error appears when you use the project monitor API.
+====
+
+[discrete]
+[[breaking-147616]]
+.Removed the `current_upgrades` endpoint. (8.7)
+[%collapsible]
+====
+*Details* +
+The `api/fleet/current_upgrades` endpoint has been removed. For more information, refer to {kibana-pull}147616[#147616].
+
+*Impact* +
+When you upgrade to 8.7.0, use the `/action_status` endpoint.
```

Comment:
I'm not sure that `/action_status` is actually the right path. Is it `/agents/action_status`? Perhaps @kpollich or someone from the Fleet team could confirm.

Reply:
```suggestion
When you upgrade to 8.7.0, use the `api/fleet/agents/action_status` endpoint.
```
Looked it up in our API docs https://www.elastic.co/docs/api